docs/limitations.md

```diff
@@ -41,6 +41,21 @@ Mutant cannot emit mutations for some subjects.
     end
     ```
 
+* multiline-style singleton methods not defined on `self`
```

Comment:
grouping by `inline-style` and `multiline-style` seems like a confusing distinction to me.